python/lsst/analysis/drp/skyPlot.py

```diff
@@ -262,11 +270,39 @@ def skyPlot(self, catPlot, plotInfo, sumStats):
             bbox = dict(facecolor="C0", alpha=0.3, edgecolor="none")
             ax.text(0.8, 0.91, starStatsText, transform=fig.transFigure, fontsize=8, bbox=bbox)
 
+        if np.any(catPlot["sourceType"] == 10):
+
+            statAll = (catPlot["useForStats"] == 1)
+            statAllMed = np.nanmedian(catPlot.loc[statAll, zCol])
+            statAllMad = sigmaMad(catPlot.loc[statAll, zCol], nan_policy="omit")
+
+            allStatsText = ("Median: {:0.2f}\n".format(statAllMed) + r"$\sigma_{MAD}$: "
+                            + "{:0.2f}\n".format(statAllMad) + r"n$_{points}$: "
+                            + "{}".format(len(catPlot)))
+            bbox = dict(facecolor="purple", alpha=0.2, edgecolor="none")
+            ax.text(0.8, 0.91, allStatsText, transform=fig.transFigure, fontsize=8, bbox=bbox)
+
+        if np.any(catPlot["sourceType"] == 9):
+
+            statAll = (catPlot["useForStats"] == 1)
+            statAllMed = np.nanmedian(catPlot.loc[statAll, zCol])
+            statAllMad = sigmaMad(catPlot.loc[statAll, zCol], nan_policy="omit")
+
+            allStatsText = ("Median: {:0.2f}\n".format(statAllMed) + r"$\sigma_{MAD}$: "
+                            + "{:0.2f}\n".format(statAllMad) + r"n$_{points}$: "
+                            + "{}".format(len(catPlot)))
+            bbox = dict(facecolor="green", alpha=0.2, edgecolor="none")
+            ax.text(0.8, 0.91, allStatsText, transform=fig.transFigure, fontsize=8, bbox=bbox)
+
         toPlotList = []
         if np.any(catPlot["sourceType"] == 1):
             toPlotList.append((xsStars, ysStars, colorValsStars, "winter_r", "Stars"))
-        elif np.any(catPlot["sourceType"] == 2):
+        if np.any(catPlot["sourceType"] == 2):
```

Comment:
These could all be `elif`, and then you could log something at the end if there's anything left that's errantly corresponding to any other number.

Reply:
These are all if deliberately so that it will plot all the different source types onto one plot if you want it to.